doc/nrf/releases_and_maturity/releases/release-notes-3.1.0.rst

```diff
@@ -0,0 +1,1190 @@
+    The section describes how to select the networking layer for Matter applications.
+
+* Updated:
+
+  * The ``west zap-generate`` command to remove previously generated ZAP files before generating new files.
+    To skip removing the files, use the ``--keep-previous`` argument.
+  * The :ref:`ug_matter_creating_custom_cluster` user guide by adding information about implementing custom commands.
+
+Matter fork
++++++++++++
+
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, the ``v1.4.2.0`` tag.
+
+The following list summarizes the most important changes inherited from the upstream Matter:
+
+* Updated:
+
+  * Improved robustness in group session management.
+  * Optimized the device commissioning process.
+
+* Fixed incorrect memory releases and unhandled exceptions.
+
+nRF IEEE 802.15.4 radio driver
+------------------------------
+
+* Added the Kconfig options to configure default CSMA-CA algorithm parameters (:kconfig:option:`CONFIG_NRF_802154_CSMA_CA_MIN_BE_DEFAULT`, :kconfig:option:`CONFIG_NRF_802154_CSMA_CA_MAX_BE_DEFAULT`, :kconfig:option:`CONFIG_NRF_802154_CSMA_CA_MAX_CSMA_BACKOFFS_DEFAULT`).
+* Updated the Kconfig option :kconfig:option:`CONFIG_NRF_802154_CCA_ED_THRESHOLD` has been replaced by :kconfig:option:`CONFIG_NRF_802154_CCA_ED_THRESHOLD_DBM` to ensure consistent behavior on different SoC families and to reduce the likelihood of misconfiguration.
+
+Thread
+------
+
+* Added the new architecture option to use the OpenThread stack directly to communicate with the IEEE 802.15.4 radio driver.
+  See the :ref:`openthread_stack_architecture` user guide for more information.
+  The new architecture option reduces the memory footprint of the OpenThread stack by around 4% and the RAM usage by around 12% in the :ref:`ot_cli_sample` sample.
+
+Wi-Fi®
+------
+
+* Added support for EAP-PEAP and EAP-TTLS authentication methods to enterprise security in the Wi-Fi management API.
+
+Applications
+============
+
+|no_changes_yet_note|
+
+Connectivity bridge
+-------------------
+
+* Fixed to resume Bluetooth connectable advertising after a disconnect.
+
+
+IPC radio firmware
+------------------
+
+|no_changes_yet_note|
+
+Matter bridge
+-------------
+
+* Implemented the missing identify cluster for the endpoint 1.
+  This resolves the :ref:`known issue <known_issues>` KRKNWK-20019.
+
+nRF5340 Audio
+-------------
+
+* Added:
+
+  * Experimental support for Audio on the nRF5340 DK, with LED state indications and button controls.
+  * Experimental Support for stereo in :ref:`broadcast sink app<nrf53_audio_broadcast_sink_app>`.
+    The broadcast sink can now receive audio from two BISes and play it on the left and right channels of the audio output, if the correct configuration options are enabled.
+    The I2S output will be stereo, but :zephyr:board:`nrf5340_audio_dk` will still only have one audio output channel, since it has a mono codec (CS47L63).
```

Comment:
Please add to the same point:
RTT shell commands have been added to select if the HW codec shall play back I2S channel 0/left, 1/right or 0 and 1 mixed (default).